crates/typst/src/geom/stroke.rs

```diff
@@ -382,6 +383,11 @@ impl Resolve for DashLength {
 
 cast! {
     DashLength,
+    self => match self {
+        Self::LineWidth => "dot".into_value(),
+        Self::Length(l) => l.into_value(),
+    },
+
```

Comment:
Might wanna take a look at this and confirm it's OK. Note that obtaining the actual value on the `LineWidth` case isn't possible in the generic case... But perhaps we can automatically convert the `"dot"` to the stroke's `thickness` when getting its `.dash`? Probably not though

Reply:
Not much we can do here I think.